katas/content/multi_qubit_gates/index.md

```diff
@@ -609,9 +627,9 @@ Let's apply the anti-controlled $X$ gate to the $|00\rangle$ state step by step:
 
 1. Transform the state of the control qubit to $|1\rangle$: we can do that by applying the $X$ gate to the first qubit:
 $$|00\rangle \rightarrow |10\rangle$$
-2. Apply the regular $CNOT$ gate:
+1. Apply the regular $CNOT$ gate:
 $$|10\rangle \rightarrow |11\rangle$$
-3. Now, if we don't undo the change we did on the first step, we'll end up with a gate that transforms $|00\rangle$ into $|11\rangle$, which is not the transformation we're trying to implement.
+1. Now, if we don't undo the change we did on the first step, we'll end up with a gate that transforms $|00\rangle$ into $|11\rangle$, which is not the transformation we're trying to implement.
```

Comment:
I understand that this wouldnt affect rendering of a numbered list, markdown is smart enough :).
It would be helpful if these changes could be reverted for cleaner diff and better readability?

Reply:
+1 :-)